app/samples/shield_mcp3561r/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.13.1)
```

Comment:
For some reason half this sample got wiped out (maybe during the reorganization after irec) so im adding it back